src/components/Button/Button.module.css

```diff
@@ -1,4 +1,4 @@
-.button {
+button {
```

Comment:
I think we should not be using element selectors, as it will apply to all buttons in our project